assets/js/vue/components/information-product/InformationProductForm.vue

```diff
@@ -321,6 +345,70 @@ export default {
     this.populateResearchGroups();
   },
 };
+
+let myDropzone;
+
+const addFileToInformationProduct = (file, done) => {
+  const currentFile = file;
+  let fileName = '';
+  if (currentFile.fullPath) {
+    fileName += currentFile.fullPath ?? currentFile.name;
+  } else if (currentFile.webkitRelativePath) {
+    fileName += currentFile.webkitRelativePath;
+  } else {
+    fileName += currentFile.name;
+  }
+  const chunkData = {};
+  chunkData.dzuuid = file.upload.uuid;
+  chunkData.dztotalchunkcount = file.upload.totalChunkCount;
+  chunkData.fileName = fileName;
+  chunkData.dztotalfilesize = file.size;
+  postApi(
+    // eslint-disable-next-line no-undef
+    `${Routing.generate('pelagos_api_add_file_information_product')}`,
+    chunkData,
+  ).then((response) => {
+    const fileId = response.data.id;
+    file.fileId = fileId;
+    thisComponent.form.file = fileId;
```

Comment:
If the file is removed after uploading, we need to set it to null again

Reply:
Worse than that! We should also delete it from disk. Glad you noticed it too.